main.js

```diff
@@ -0,0 +1,212 @@
+		icon.classList.add('icon', 'material-symbols-outlined');
+		icon.setAttribute("name", cards[x]);
+		icon.innerText = cards[x];
+		back.appendChild(icon);
+		cardContainer.appendChild(front);
+		cardContainer.appendChild(back);
+		fragment.appendChild(cardContainer);
+	}
+	board.appendChild(fragment);
+	console.log(cards);
+}
+
+
+//-----CHOOSE DIFFICULTY
+function chooseDifficulty(){
+	[].forEach.call(difficulties, function(input){
+		input.nextElementSibling.classList.remove('checked');
+		console.log(input.nextElementSibling)
+		if (input.value === 'easy' && input.checked === true) {
+			NumberOfCards = 12;
+			difficulty = 'easy';
+			input.nextElementSibling.classList.add('checked');
+			board.classList.remove('normal-game', 'hard-game');
+			board.classList.add('easy-game');
+		} else if (input.value === 'normal' && input.checked === true) {
+			NumberOfCards = 24;
+			difficulty = 'normal';
+			input.nextElementSibling.classList.add('checked');
+			board.classList.remove('easy-game', 'hard-game');
+			board.classList.add('normal-game');
+		} else if (input.value === 'hard' && input.checked === true) {
+			NumberOfCards = 36;
+			difficulty = 'hard';
+			input.nextElementSibling.classList.add('checked');
+			board.classList.remove('easy-game', 'normal-game');
+			board.classList.add('hard-game');
+		}
+	});
+}
+
+function stopwatch(){
+	sec += 1;
+	let minutes = Math.floor(sec/60);
+	let seconds = sec % 60;
+	if (sec<10) {
+		timer.innerText = "00:0"+sec;
+	}
+	else if (sec<60) {
+		timer.innerText = "00:"+sec;
+	}
+	else if (sec<3600) {
+		timer.innerText = minutes+":"+seconds;
+	}
+}
+
+// -----CHECK WIN AND DISPLAY MODAL
+function checkwin(num) {
+	let won;
+	switch (difficulty) {
+		case 'easy' :
+			if (num === 6) {
+				won = true;
+			};
+			break;
+		case 'normal' :
+			if (num === 12) {
+				won = true;	
+			};
+			break;
+		case 'hard' :
+			if (num === 18){
+				won = true;
+			};
+			break;
+	};
+	if (won === true) {
+		setTimeout(() => {
+			//fill and show modal
+			document.getElementById('final-time').innerText = timer.innerText;
+			document.getElementById('final-moves').innerText = moves;
+			modal.classList.remove('hide');
+			//stop the stopwatch
+			clearInterval(setTimer);
+		}, 500);
+	}
+}
+
+
+//-----CORE OF THE GAME - CHECKING CARDS AND COUNTING MOVES
+function matchChecker(event){
+	//making sure the clicked target is a card and prevent doubleclicking
+	if (event.target.classList.contains('card') && !event.target.classList.contains('front-open')) {
+		//flip the card on click
+		event.target.classList.add('front-open');
+		event.target.nextElementSibling.classList.add('back-open');
+		//keep track of the attribute name (card name) of the clicked cards
+		cardMatch.push(event.target.nextElementSibling.firstChild.getAttribute("name"));
+		selectedCards.push(event.target);
+		moveCount += 1;
+		//allow only two cards to be selected, then verify if they match match
+		if (moveCount === 2) {
```

Comment:
Și aici poate fi inversat `if`-ul, din același motiv.

```suggestion
		if (moveCount < 2) {
		    return;
		}
```